dspace-spring-rest/src/test/java/org/dspace/app/rest/test/AbstractEntityIntegrationTest.java

```diff
@@ -0,0 +1,77 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.rest.test;
+
+import org.dspace.app.rest.builder.EntityTypeBuilder;
+import org.dspace.app.rest.builder.RelationshipTypeBuilder;
+import org.dspace.content.EntityType;
+import org.junit.Before;
+
+public class AbstractEntityIntegrationTest extends AbstractControllerIntegrationTest {
+
+
+    @Before
+    public void setUp() throws Exception {
```

Comment:
Because this is creating a lot of content, it'd be nice to add a comment to describe the Entities being created here.

Reply:
We'll add some docs describing what this function is doing. It is basically building the contents of relationship-types.xml